crypto/bls12/bls12.go

```diff
@@ -86,37 +91,8 @@ func (priv *PrivateKey) Public() consensus.PublicKey {
 	return &PublicKey{p: bls12.NewG1().MulScalarBig(p, &bls12.G1One, priv.p)}
 }
 
-// Signature is a bls12-381 signature.
-type Signature struct {
-	signer hotstuff.ID
-	s      *bls12.PointG2
-}
-
-// ToBytes returns the object as bytes.
-func (s *Signature) ToBytes() []byte {
-	var idBytes [4]byte
-	binary.LittleEndian.PutUint32(idBytes[:], uint32(s.signer))
-	// not sure if it is better to use compressed or uncompressed here.
-	return append(idBytes[:], bls12.NewG2().ToCompressed(s.s)...)
-}
-
-// FromBytes unmarshals a signature from a byte slice.
-func (s *Signature) FromBytes(b []byte) (err error) {
-	s.signer = hotstuff.ID(binary.LittleEndian.Uint32(b))
-	s.s, err = bls12.NewG2().FromCompressed(b[4:])
-	if err != nil {
-		return fmt.Errorf("bls12: failed to decompress signature: %w", err)
-	}
-	return nil
-}
-
-// Signer returns the ID of the replica that generated the signature.
-func (s *Signature) Signer() hotstuff.ID {
-	return s.signer
-}
-
-// AggregateSignature is a bls12-381 aggregate signature. The participants map contains the IDs of the replicas who
-// participated in the creation of the signature. This allows us to build an aggregated public key to verify the
+// AggregateSignature is a bls12-381 aggregate signature. The participants bitmap contains the IDs of the replicas that
```

Comment:
Just to clarify, do you mean `The participant's bitmap` or the plural form??

Reply:
I mean the `participants` field in the struct.